.github/workflows/create-image-without-release.yml

```diff
@@ -0,0 +1,172 @@
+# This workflow will release project with Gradle
+name: Create Image Only (No Release)
+
+on:
+    push:
+
```

Comment:
Not seeing any use of a specific branch to build the containers?

Reply:
will be done once we gonna use the workflow_dispatch instead of push